test/unit/helpers/QtiPackageExporterTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\test\unit\helpers;
+
+use common_Exception;
+use common_report_Report;
+use oat\oatbox\filesystem\Directory;
+use oat\oatbox\filesystem\File;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\tao\helpers\FileHelperService;
+use taoQtiTest_models_classes_export_TestExport22;
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\taoQtiTest\helpers\QtiPackageExporter;
+
+class QtiPackageExporterTest extends TestCase
```

Comment:
Your test cases are not asserting on the expected positive behaviour of your service:
- What is filename?
- What is the filepath?
- What about the file content?

Reply:
I think first two cases should be testes in test for FileSystemService. Here if I create a mock for File object it will return filename and filepath which will be configured for this mock. 
We can test that `getDirectory` and `getFile` methods of a mock will be called with expectedvalues. 

Regarding file content - it's generated in another class. For the moment added assertion that `File::put()` method will be called with expected content returned by another mock. But I need to check if local file system can be registered to eliminate all  this logic (at the moment related method is marked as deprecated).